### DISCUSSION TITLE
Build on Seperate System

### DISCUSSION THREAD

**Discussion by TKK13909** [Bali10050/Darkly#3] [Ideas]
I run an Atomic Fedora image which means I cannot simply build this theme on my system using the regular Fedora installation method. What would I need to do after building this in a Distrobox container to add this theme to my system?

**Comment by TKK13909** [comment]
Once this is figured out we could probably start working on making the theme automatically build using Github CI/CD.

**Reply by DeltaCopy** [reply]
I am working on setting up a POC build job over on the Fedora Copr.
Depending on how things go the test would be to enable the copr, and install via dnf

**Reply by TKK13909** [reply]
I'm not super familiar with building KDE Plasma themes using cmake so forgive me if this is an ignorant question. Is there a way to build this theme as a standalone and then put it into the .local/share/plasma/desktoptheme to install it like with other themes from store.kde.org? I just need a way to build and install it locally without messing with any root directories. If this is not currently possible, I can use a second drive with regular Fedora to build and test this theme so I can contribute.

**Reply by Bali10050** [reply]
This currently doesn't include a desktop theme, and I'm currently mostly working on the application theme for qt, and I haven't really heard about any qt themes that you can install without root. If you're intrested in contributing with a plasma theme, I have made [my own](https://www.pling.com/p/1595609/) years ago, and it really needs a refresh at the moment, so I would be happy if you made a fork for it here on github, and if I figure out how to do it, I could include it with this theme.

**Reply by TKK13909** [reply]
I just submitted a PR for installing this theme using distrobox.

**Comment by TKK13909** [comment]
Ah ok. Now I understand. I just messed about for a bit and got it installed to $HOME/.local/lib64/plugins. Running `export QT_PLUGIN_PATH=$HOME/.local/lib64/plugins:$QT_PLUGIN_PATH` and then running a QT application does it. I'll put together some instructions that use distrobox and then submit a PR.
